third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -1900,7 +1950,7 @@ fn parse_dot_or_index_chain(context: &mut Context) -> Result<Exp, Box<Diagnostic
         let exp = match context.tokens.peek() {
```

Comment:
Commenting here because it is visible close by. In the documentation above, we say "spec only" for something that is no longer spec only and therefore should be removed.

Reply:
done